### DISCUSSION TITLE
TMRP Main Discussion

### DISCUSSION THREAD

**Discussion by nasko222** [nasko222/the-minecraft-recreation-project#92] [General]
## 👋 Welcome!
 
Welcome to The Minecraft Recreation Project's Discussion Page. Here you can share images of gameplay or ask stuff about the project, idk. Have fun I guess

**Comment by minecraft12y** [comment]
Hey, this is pretty cool

**Comment by ghost** [comment]
Beta 1.8 Dev 3
![2023-04-24_10 19 24](https://user-images.githubusercontent.com/62471032/234056768-b82c9ecb-e82d-4d28-968b-350b124c8d89.png)

**Reply by TheBlank860** [reply]
seed?

**Reply by TechGuysTechSpotOfficial** [reply]
that's an outdated remake, check out the hybrid thread for the most up to date beta 1.8 remakes

**Reply by RossGamer2048** [reply]
an old village

**Comment by SeaGlowingPro** [comment]
One thing. Should there be recreations of Notch's early games, before he made Minecraft?

I already have a W.I.P. remake of 3D1, using the known build of Legend of the Chambered as a base.
![3d1_remake](https://user-images.githubusercontent.com/126473525/234252686-28301d3a-31b5-4b3d-9ac1-db1e39bcf9fb.png)

**Reply by nasko222** [reply]
> One thing. Should there be recreations of Notch's early games, before he made Minecraft?
> 
> I already have a W.I.P. remake of 3D1, using the known build of Legend of the Chambered as a base. ![3d1_remake](https://user-images.githubusercontent.com/126473525/234252686-28301d3a-31b5-4b3d-9ac1-db1e39bcf9fb.png)

Maybe not in TMRP, But you can of course make a new repo, I give a name suggestion I guess Notch Games Recreation Project (NGRP ??) IDK

Infact it was method who suggested the name "The Minecraft Recreation Project" Before that I was going with "methodarchive"

**Reply by Vector-Graphics** [reply]
hi

**Comment by nasko222** [comment]
dev10, forest village near spawn seed
![image](https://user-images.githubusercontent.com/52720373/235313440-02525180-9f57-4603-9492-bebbbd329831.png)

**Reply by Spicytomatoes777** [reply]
what abt dev2 or dev3

**Comment by BlueStaggo** [comment]
I discovered a potential developer version myself, which I dub 12w03a-dev and 12w03a-dev2. Both of these versions were shown by Jeb and feature early Jungle trees with oak leaves and spruce trunks, although these could just be early jungle log and leaves textures. The large trees in the first dev version have a more bulbous shape and lacked branches. These are found in [this tweet](https://twitter.com/jeb_/status/159616418575421440) and [that tweet](https://twitter.com/jeb_/status/159940561418199041) respectively.

![12w03a-dev](https://i.imgur.com/XaVJu.png)
![12w03a-dev2](https://i.imgur.com/Li3Ak.png)

 One funny thing I discovered is that jungles in Minecraft were shown off around the same time jungles in Cube World were shown off:
 
![Wolfram von Funck @wol_lay: "New Cube World Screenshots: Jungle goo.gl/fb/VwuBw"; Jens Bergensten @jeb_: "@wol_lay
 Haha, and I was just doing this: http://i.imgur.com/XaVJu.png"](https://github.com/nasko222/the-minecraft-recreation-project/assets/71090710/64b2b4ae-0315-4065-be75-6b6c1e29bbda)

**Reply by CaptainDumb** [reply]
wait, i had the same world on 12w03a

**Comment by nasko222** [comment]
Nice find!

**Comment by nasko222** [comment]
dead chat

![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/52720373/b0a2f2f6-ee7b-46b0-9e75-755d97af0c95)

**Comment by bunnybeep** [comment]
b1.8 dev2 
seed: gargamel
![2023-09-06_00 05 15](https://github.com/nasko222/the-minecraft-recreation-project/assets/83859658/613fd648-7f7f-40d3-9a7c-c2860368f984)
![2023-09-06_00 05 22](https://github.com/nasko222/the-minecraft-recreation-project/assets/83859658/61b48fa2-898b-4e25-aa77-bcfce620d5f6)

**Reply by Gianni46363** [reply]
woah

**Comment by G23M08** [comment]
How can I help also it wont work on multi mc or beta craft I have to use mine online

**Reply by nasko222** [reply]
help with dev or what exactly?

playtesting and finding inacurracies would be helpful

i dont look for more dev help for now

**Reply by nasko222** [reply]
it doesnt work in mmc and betacraft what? what even is mineonline, i havent heard anyone else having the issues before. Did you carefully follow the tutorial?

**Reply by G23M08** [reply]
mine online is another old minecr5aft thing and yes I did it and it still crashes on Multi mc and doesn't show up in beta craft

**Reply by nasko222** [reply]
sorry, idk how to help, i never had anyone having that issue.

**Reply by G23M08** [reply]
and playtesting would be good for me

**Reply by G23M08** [reply]
if I copy the crash output will it help?

**Reply by nasko222** [reply]
send it, ill give it a look when i have time

**Reply by bunnybeep** [reply]
It doesn't seem to work on mineonline from my experience, but for it to work in betacraft you just pop the jar files into the versions folder in (name)\AppData\Roaming\.betacraft\versions 
you may have to have previously launched the base release version your trying to play and it seems to be funky if you use them in a separate instance aside from the base instance, though this could just me an error on my end, hope this helps! ^^
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/83859658/eac0d6ce-52fa-4f15-8e3a-4a2c952a4829)

![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/83859658/129717b5-2f22-4a4c-ac8d-1a3eb8b158e0)

**Reply by G23M08** [reply]
change the betacreaft tutorial you dont need the info files

**Reply by G23M08** [reply]
MultiMC version: 0.7.0-3813


Launched instance in online mode

authserver.mojang.com resolves to:
    [108.139.95.120, 108.139.95.17, 108.139.95.22, 108.139.95.79]

session.minecraft.net resolves to:
    [2620:1ec:bdf::33, 2620:1ec:46::33, 13.107.246.33, 13.107.213.33]

textures.minecraft.net resolves to:
    [2620:1ec:46::33, 2620:1ec:bdf::33, 13.107.246.33, 13.107.213.33]

api.mojang.com resolves to:
    [2620:1ec:46::34, 2620:1ec:bdf::34, 13.107.213.34, 13.107.246.34]


Minecraft folder is:
E:/Users/algec/Downloads/mmc-develop-win32 (1)/MultiMC/instances/rd-1613481/.minecraft


Java path is:
C:/Program Files (x86)/Common Files/Oracle/Java/javapath/javaw.exe


Checking Java version...
Java is version 1.8.0_361, using 64-bit architecture, from Oracle Corporation.


Main Class:
  com.mojang.minecraft.RubyDung

Native path:
  E:/Users/algec/Downloads/mmc-develop-win32 (1)/MultiMC/instances/rd-1613481/natives

Traits:
traits no-texturepacks

Libraries:
  E:/Users/algec/Downloads/mmc-develop-win32 (1)/MultiMC/libraries/net/java/jinput/jinput/2.0.5/jinput-2.0.5.jar
  E:/Users/algec/Downloads/mmc-develop-win32 (1)/MultiMC/libraries/net/java/jutils/jutils/1.0.0/jutils-1.0.0.jar
  E:/Users/algec/Downloads/mmc-develop-win32 (1)/MultiMC/libraries/org/lwjgl/lwjgl/lwjgl/2.9.4-nightly-20150209/lwjgl-2.9.4-nightly-20150209.jar
  E:/Users/algec/Downloads/mmc-develop-win32 (1)/MultiMC/libraries/org/lwjgl/lwjgl/lwjgl_util/2.9.4-nightly-20150209/lwjgl_util-2.9.4-nightly-20150209.jar
  E:/Users/algec/Downloads/mmc-develop-win32 (1)/MultiMC/instances/rd-1613481/.minecraft/bin/minecraft.jar

Native libraries:
  E:/Users/algec/Downloads/mmc-develop-win32 (1)/MultiMC/libraries/net/java/jinput/jinput-platform/2.0.5/jinput-platform-2.0.5-natives-windows.jar
  E:/Users/algec/Downloads/mmc-develop-win32 (1)/MultiMC/libraries/org/lwjgl/lwjgl/lwjgl-platform/2.9.4-nightly-20150209/lwjgl-platform-2.9.4-nightly-20150209-natives-windows.jar

Jar Mods:
  pc-131648 (e8eab39e-c72a-4481-b8a5-25635ae63f02.jar)

Params:
  

Window size: 854 x 480

Java Arguments:
[-Dfml.ignoreInvalidMinecraftCertificates=true, -Dfml.ignorePatchDiscrepancies=true, -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump, -Xms512m, -Xmx1024m, -Duser.language=en]


Minecraft process ID: 8016


Using onesix launcher.

java.io.FileNotFoundException: level.dat (O sistema não pode encontrar o arquivo especificado)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at com.mojang.minecraft.level.Level.load(Level.java:79)
	at com.mojang.minecraft.level.Level.<init>(Level.java:29)
	at com.mojang.minecraft.RubyDung.init(RubyDung.java:99)
	at com.mojang.minecraft.RubyDung.run(RubyDung.java:127)
	at java.lang.Thread.run(Unknown Source)
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007fff6c062ee7, pid=8016, tid=0x0000000000001480
#
# JRE version: Java(TM) SE Runtime Environment (8.0_361) (build 1.8.0_361-b09)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.361-b09 mixed mode windows-amd64 compressed oops)
# Problematic frame:
# C  [ig7icd64.dll+0x22ee7]
#
# Failed to write core dump. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# E:\Users\algec\Downloads\mmc-develop-win32 (1)\MultiMC\instances\rd-1613481\.minecraft\hs_err_pid8016.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Process exited with exit code 1 (0x1).
Please note that usually neither the exit code, nor its description are enough to diagnose issues!
Always upload the entire log and not just the exit code.

**Reply by G23M08** [reply]
also the 3.1 devs dosnt work

**Reply by RossGamer2048** [reply]
also with me when i launching pc-20090515 with multi mc
@G23M08 did you say 3.1?
ABSOLUTE CINEMA

**Comment by nasko222** [comment]
windows 7, youre a keeper. be careful of the new webp exploit tho

**Comment by CaptainDumb** [comment]
i always convert them images into png since webp fucking sucks :skull:

**Comment by G23M08** [comment]
what is the best way to play these versions? like what launcher

**Comment by nasko222** [comment]
betacraft

**Reply by G23M08** [reply]
i use beta craft and it stilll crashes

**Comment by nasko222** [comment]
for other launchers use the following jvm args -Dhttp.proxyHost=betacraft.uk -Djava.util.Arrays.useLegacyMergeSort=true

**Comment by Illia24** [comment]
You will spawn in house (village)
Seed: 3798420633838731088

**Reply by nasko222** [reply]
version?

**Reply by Illia24** [reply]
Also dev build: beta 1.8 preview

**Reply by Illia24** [reply]
I found this seed somehow cool because of spawn place ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯

**Comment by G23M08** [comment]
In the hybrid 1.8 beta version will there be buttons like 
Build 1 date xx/xx/xx
Build 2date
So on so on?

**Comment by nasko222** [comment]
No, they will be pre-compiled, that gui is only for nerds who want to debug

it will be normal jar files for each build

**Reply by G23M08** [reply]
Oh ok didn't understand What you said about it
Also can I test the test 1.8 things?

**Reply by G23M08** [reply]
is it possible for you to make a JSON file so its possible to use this on the Minecraft launther

**Comment by JONESTHEBIRD** [comment]
![Screenshot 2024-03-24 080631](https://github.com/nasko222/the-minecraft-recreation-project/assets/124440535/7f24bbc1-a381-451b-b769-2759edf44305)
I don't think this is that laggy.

**Reply by nasko222** [reply]
> 
> ![Screenshot 2024-03-24 080631](https://github.com/nasko222/the-minecraft-recreation-project/assets/124440535/7f24bbc1-a381-451b-b769-2759edf44305)
> I don't think this is that laggy.
> 

you got 3060, are you expecting 2 fps?

**Reply by JONESTHEBIRD** [reply]
I just wondered how many fps I get because SeaGlowingPro got 150 fps only

**Reply by nasko222** [reply]
i get 300 fps cuz ive capped my java, it could also be gsync/vsync

**Reply by JONESTHEBIRD** [reply]
ok, this is why then

**Comment by 483aa** [comment]
What about moving cave gen on pc-131648?? I moved cg on pc-48, 41, 1208 and 1151.

**Reply by nasko222** [reply]
as i said i no longer care for speculative builds and anything before 131655

**Comment by JONESTHEBIRD** [comment]
I have played pc-131655 for 6 hours..... I need a saving mod. Is there any way making a saving mod without having to modify any existing classes in the remake and use it as a jar mod?

**Comment by nasko222** [comment]
no its not, but theres a way to make saving mod

**Comment by JONESTHEBIRD** [comment]
the ones who have the code can make one. You just have to copy a few lines of code from pc-132011 if I'm right

**Comment by JONESTHEBIRD** [comment]
I think I know what I have to do - wait. Notch, why... why is byte blocks private????

**Comment by JONESTHEBIRD** [comment]
well this won't work :(

**Comment by JONESTHEBIRD** [comment]
![Screenshot 2024-03-25 135226](https://github.com/nasko222/the-minecraft-recreation-project/assets/124440535/017bb6cf-de32-4c2b-915e-83cba303eeec) uhm.. #1 pc-131655 player?

**Reply by nasko222** [reply]
> ![Screenshot 2024-03-25 135226](https://github.com/nasko222/the-minecraft-recreation-project/assets/124440535/017bb6cf-de32-4c2b-915e-83cba303eeec) uhm.. #1 pc-131655 player?
> 

any pc player should work

**Reply by JONESTHEBIRD** [reply]
wow... I can decompile mods in retromcp

**Reply by nasko222** [reply]
> wow... I can decompile mods in retromcp

you found that out now?

**Reply by JONESTHEBIRD** [reply]
Yes
I was scrolling through the wiki

**Reply by JONESTHEBIRD** [reply]
I can make a saving mod for myself on my own Ig

**Reply by JONESTHEBIRD** [reply]
I only have to wait now..

**Reply by JONESTHEBIRD** [reply]
and wait

**Reply by JONESTHEBIRD** [reply]
Yes, I managed to do a saving mod. With a loading and saving method. Only loading does not work XDD

**Comment by TechGuysTechSpotOfficial** [comment]
My Pre-Classic page is really outdated
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/caab9763-94ee-4ed0-a141-debd1ba9b1ce)

**Reply by TechGuysTechSpotOfficial** [reply]
Actually most of my multimc instances are outdated

**Reply by JONESTHEBIRD** [reply]
You should make a own group for the speculations and uptdate the outdated remakes

**Reply by TechGuysTechSpotOfficial** [reply]
yea good idea

**Reply by RossGamer2048** [reply]
I have got pc-10-pre, pc-11, pc-12, pc-131137 (dont working), pc-131151, pc-131208, pc-131641, pc-131648, pc-131655, pc-131825, and pc-131951. Also i have got pc-20090515 but it's isn't working

**Reply by mattkmccollum-wq** [reply]
wait, I'm new. how did you get those? the oldest version I have is pc-131648

**Reply by nasko222** [reply]
these are based only on speculations and thus been removed from TMRP

**Reply by mattkmccollum-wq** [reply]
do these versions still exist tho? like can I still download them somewhere?

**Comment by G23M08** [comment]
i have an idea a wiki showing the differentces from these dev versions to normal ones

**Reply by Christo7899** [reply]
I have a google sheets that does essentially the same thing. 

https://docs.google.com/spreadsheets/d/1WV1Xkw203POcEYmi3Tp-AYlCYAO-NOi2vFkYELPDhmk/edit?usp=sharing

**Comment by G23M08** [comment]
does  0.0.3a and 0.0.9a crash on the asus eee?

**Reply by nasko222** [reply]
Guys I know you expect accuracy from this project but... seriously?

**Reply by nasko222** [reply]
I don't know actually though

**Reply by G23M08** [reply]
No I was just kidding

**Reply by G23M08** [reply]
Like how would u make it crash on one specific computer that no one uses anymore

**Comment by sigma2424243** [comment]
does anyone know how to stop pre classic versions of minecraft from crashing right away. Like it works for a split second and then crashes. Im on betacraft btw

**Reply by JONESTHEBIRD** [reply]
remove the level.dat in the folder

**Comment by somepersononthenet** [comment]
beta 1.9 prerelease 4 dev7
![Minecraft b1 9-pre4-1435 8_19_2024 8_29_33 PM](https://github.com/user-attachments/assets/fecbe69e-b922-43f4-9b6d-59d26dd08c1a)

**Comment by CrumbledHam** [comment]
Is there a source for the versions.

**Comment by Spicytomatoes777** [comment]
hey you know what rd-20090515 is in pc? i want to figure it out becuase it isnt pc-152225

**Reply by SeaGlowingPro** [reply]
We do not know for sure, as there isn't exact proof that the May 15th build (rd-20090515) was compiled or playtested at a specific time. That's why it's left out as "pc-20090515" in the pre-Classic branch.

**Reply by Spicytomatoes777** [reply]
thanks for the info

**Comment by Spicytomatoes777** [comment]
someone should make a server out of these versions like the dev versions to beta 1.8 like beta 1.8 dev2 or dev9

**Reply by Spicytomatoes777** [reply]
but itll probably never happen

**Reply by BlueStaggo** [reply]
Fun fact: when I was reworking the Beta 1.8 devs before the Hybrid started development, I actually did make server versions of the Beta 1.8 devs that I had already worked on at that point. Me, Nasko and a few other people played on a private server for a bit on the river test build. Sure was cool, but I was just playing around.

**Reply by Spicytomatoes777** [reply]
really? wish there were servers of beta 1.8 devs that werent private

**Reply by nasko222** [reply]
you didnt miss out anything big really, those b1.8 dev servers were heavily unfinished and missing out important features, i dont like to promise things but i may attempt making "hybrid" server as well, in the future, currently ive got next to no free time to work on anything online

**Reply by Spicytomatoes777** [reply]
ok

**Comment by nasko222** [comment]
Hey everyone, our TMRP team has made a public TMRP (And The Hybrid) discord server for easier discussions about the project, sharing your ideas/feedback and other stuff, the server is still new and if this idea catches on it will be moderated properly in the future

Invite link - https://discord.gg/jQxU7c3k4X

**Comment by RossGamer2048** [comment]
my game with pc-131137:
RLY GUD SHIT BRO

**Comment by nasko222** [comment]
Writing a statement on my own about the future of TMRP. Currently real life work has been extensively hard to the point I cannot use my computer for more than a week, this has never happened before. I want to point out that TMRP and The Hybrid are not cancelled in any way and I don't want people to believe that and disregard their interest to the project. I know this message will be sent on the Discord server as well (and I have intent to as I want people there to read it too) and I want to spread awareness and clear confusion to those who may believe the project won't be continued or finished (Both TMRP and Hybrid). Currently I cannot give a clear estimate on when I will "land on my feet" with my real life work and all I can say is that I may not be fully back until the end of the year. Also if I do in fact have free time to work on the project, which will be very little, I wouldn't want this to be taken as me coming back to actively work like I used to or be expected to work. I know most of the users are young and have less patience than usual so I wanted to clear out that as well. For those who know me, I don't leave stuff unfinished and I have interests to work on TMRP in the future. Hope you all understand.
